server/container_remove.go

```diff
@@ -54,6 +55,16 @@ func (s *Server) removeContainerInPod(ctx context.Context, sb *sandbox.Sandbox,
 		if err := s.stopContainer(ctx, c, int64(10)); err != nil {
```

Comment:
I think stopContainer should have `s.nri.stopContainer()` so it doesn't have to be manually called below

Reply:
There is one code path where the control flow is NRI ingress -> Server.stopContainer() and that prevents me from simply folding unconditionally the NRI one into the corresponding CRI one like that.